generic3g/vertical/FixedLevelsVerticalGrid.F90

```diff
@@ -79,5 +91,22 @@ logical function can_connect_to(this, src, rc)
        _UNUSED_DUMMY(src)
     end function can_connect_to
 
+    logical function equal_FixedLevelsVerticalGrid(a, b) result(equal)
```

Comment:
```suggestion
    impure elemental logical function equal_FixedLevelsVerticalGrid(a, b) result(equal)
```

Reply:
These equality operations often find themselves inside array expressions.   Probably not in this case, and I'd not ask for the change were it not for the request to add more unit tests.

Reply:
done